be/test/exec/json_scanner_test2.cpp

```diff
@@ -0,0 +1,426 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "exec/broker_scan_node.h"
+
+#include <string>
+#include <map>
+#include <vector>
+
+#include <gtest/gtest.h>
+#include <time.h>
+#include "common/object_pool.h"
+#include "runtime/tuple.h"
+#include "exec/local_file_reader.h"
+#include "exprs/cast_functions.h"
+#include "runtime/descriptors.h"
+#include "runtime/exec_env.h"
+#include "runtime/runtime_state.h"
+#include "runtime/row_batch.h"
+#include "runtime/user_function_cache.h"
+#include "gen_cpp/Descriptors_types.h"
+#include "gen_cpp/PlanNodes_types.h"
+
+namespace doris {
+
+class JsonSannerTest : public testing::Test {
```

Comment:
What's the meaning of `json_scanner_test2`? Why do we both need `json_scanner_test` and `json_scanner_test2`?

Reply:
there are two senses, i had change the file name